From 9fc4bbbdd9c77c8299a015de4d1370a1b096b947 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan.agner@toradex.com>
Date: Thu, 22 Mar 2018 16:04:38 +0100
Subject: [PATCH 2/2] Remove Warnings on unknown events

Newer devices such as multitouch screens report new events which
are now known to Qt. This floods the stderr unnecessarily. Drop
the warning.

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 src/gui/embedded/qmouselinuxinput_qws.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/gui/embedded/qmouselinuxinput_qws.cpp b/src/gui/embedded/qmouselinuxinput_qws.cpp
index cb50e54448..01294b43b1 100644
--- a/src/gui/embedded/qmouselinuxinput_qws.cpp
+++ b/src/gui/embedded/qmouselinuxinput_qws.cpp
@@ -208,9 +208,6 @@ void QWSLinuxInputMousePrivate::readMouseData()
         } else {
             unknown = true;
         }
-        if (unknown) {
-            qWarning("unknown mouse event type=%x, code=%x, value=%x", data->type, data->code, data->value);
-        }
     }
 }
 
-- 
2.16.2

